debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/MySqlConnectorConfig.java

```diff
@@ -1073,7 +1071,7 @@ private static int validateSnapshotLockingMode(Configuration config, Field field
             return 1;
         }
 
-        // Determine the snapshot mode defined.
+        /*// Determine the snapshot mode defined.
```

Comment:
Why is this just commenting out this code? If we want to go that route, the check should be removed completely.